src/dom/dom.js

```diff
@@ -3104,53 +3335,47 @@ class MediaElement extends p5.Element {
   }
 
   /**
-   * If no arguments are given, returns the current time of the element.
-   * If an argument is given the current time of the element is set to it.
+   * Manages the media element's playback time. Calling `myMedia.time()`
+   * returns the number of seconds the audio/video has played. Time resets to
+   * 0 when looping media restarts.
+   *
+   * The parameter, `time`, is optional. It's a number that specifies the
+   * time, in seconds, to jump to when playback begins.
    *
    * @method time
-   * @return {Number} current time (in seconds)
+   * @return {Number} current time (in seconds).
    *
    * @example
-   * <div><code>
-   * let ele;
-   * let beginning = true;
+   * <div>
+   * <code>
+   * let dragon;
+   *
    * function setup() {
-   *   //p5.MediaElement objects are usually created
-   *   //by calling the createAudio(), createVideo(),
-   *   //and createCapture() functions.
-   *
-   *   //In this example we create
-   *   //a new p5.MediaElement via createAudio().
-   *   ele = createAudio('assets/lucky_dragons.mp3');
-   *   background(250);
+   *   background(200);
+   *
    *   textAlign(CENTER);
-   *   text('start at beginning', width / 2, height / 2);
+   *   textWrap(CHAR);
+   *   text('Double-click to goto 2 seconds', 10, 10);
```

Comment:
Might need to adjust the coordinates for the text:
<img width="213" alt="image" src="https://github.com/processing/p5.js/assets/5315059/1612c30e-f5f5-4d72-85cb-4ea45f2fac56">